script.js

```diff
@@ -1,12 +1,178 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+let allEpisodes;
+let allTVShows;
+let selectedShow;
+let allGenres;
+
+// const rootEl = document.querySelector('#root');
+const searchInput = document.querySelector('#search-input');
+const searchResult = document.querySelector('#search-result');
+const selectEpisodesEl = document.querySelector('#select');
+const selectShowEl = document.querySelector('#TVShows');
+const episodesEl = document.querySelector('#episodes');
+const showsEl = document.querySelector('#tvSeries');
+
+const getEpisodeTitle = obj => 'S' + `${obj.season}`.padStart(2, 0) + 'E' + `${obj.number}`.padStart(2, 0) + ` - ${obj.name}`;
+
+const getEpisodeCard = obj => `<div class="col-md-3 m-1 card">
+  <div class="card-body card-css">
+    <h5 class="card-title text-center">${getEpisodeTitle(obj)}</h5>
+    <hr>
+    <img src="${obj.image.original}" class="card-img-top" alt="${getEpisodeTitle(obj)} image">
+    <p class="card-text">${obj.summary}</p>
+  </div>
+</div>`;
+
+const getShowCard = obj => `
+  <div class="card col-12 col-md-10 m-3 show-card g-0" onclick="fetchShowAndDisplay(${obj.id})">
```

Comment:
I am just wondering why did you decide to use this method instead of DOM?

Reply:
I could understand it better and make changes to it faster and easier